### DISCUSSION TITLE
How to attract more programmers for merkaartor's developing?

### DISCUSSION THREAD

**Discussion by stephan75** [openstreetmap/merkaartor#252] [General]
Hello @Krakonos ,
as a long time OSM contributor I also follow the evolving of merkaartor for really many many years now.
And I think I am also aware how many efforts are needed to maintain such an open source project next to your normal daily work.

By appreciating all your efforts, I have the impression that you, Krakanos, are left as the (almost) one and only developer for this project, or not?
Wouldn't it be helpful to try to attract some more OSM aware programmers who can contribute to merkaartor by solving the outstanding issues and pull requests?
If yes, in your opinion: what would be the most urgent point to put efforts in?

Does it make sense to use the well-established OSM channels to draw attention to merkaartor in these days? If yes, I can try to contribute ...
( I am NOT an programmer, so I cannot help directly.)

**Comment by Krakonos** [comment]
Hi there!
Sorry about missing the dicussion, I'm still not used to checking it. It really depends. So far, all of the contributions from other people are mostly minor fixes. Anything in the issue tracker that improves editing experience would be useful, and having help would be awesome - I currently don't have that much time to work on the project, so I'm mostly just trying to keep the lights on (and Merkaartor is still my favourite OSM editor).

Areas we need the most work on:
- OSM API interaction - only a subset is supported, and automatic splitting of download/upload process would be awesome.
- There are a few rendering glitches, where nodes are sometimes hidden behind other features. This would require some attention, but also has very steep learning curves, as the rendering engine is rather complex and a bit messy.

Also packaging for OSX and Windows is a huge burden for me and takes a lot of time for tuning and testing. Having somebody to help with that work would be excellent. Currently that requires some knowledge of the given platform(s) and a bit of scripting and CMake (though I can help with the CMake part).
